pkg/gui/controllers/branches_controller.go

```diff
@@ -259,7 +259,6 @@ func (self *BranchesController) viewUpstreamOptions(selectedBranch *models.Branc
 
 	if !selectedBranch.RemoteBranchStoredLocally() {
 		viewDivergenceItem.DisabledReason = self.c.Tr.UpstreamNotSetError
-		unsetUpstreamItem.DisabledReason = self.c.Tr.UpstreamNotSetError
```

Comment:
There should still be a disabled reason if the branch doesn't have an upstream set at all (i.e. if `IsTrackingRemote` returns false). I'd be ok with using the same error text (UpstreamNotSetError) in that case, even if it's not totally accurate.

Reply:
You're right, good catch. 

The text _is_ accurate actually since it states `"The selected branch has no upstream (or the upstream is not stored locally)"`! :)